src/redux/migrations.ts

```diff
@@ -729,4 +729,5 @@ export const migrations = {
       providers: null,
     },
   }),
+  64: (state: any) => state,
```

Comment:
argh i was so annoyed about this. this is basically typescript being very strict - since i added the `swap` type to the fee estimation, it thinks that we should add the `swap` object to the `fee.estimation` state EVEN THOUGH it would be `swap: undefined` (i.e. not even added.....)

sigh.

so i had to do this in the test schema to silence typescript, and to keep the migration in sync with the tests i added this  migration. i guess it doesn't hurt, but agree, it doesn't add functional value here...